otcextensions/osclient/smn/client.py

```diff
@@ -0,0 +1,53 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+import logging
+
+from osc_lib import utils
+
+from otcextensions import sdk
+from otcextensions.i18n import _
+
+LOG = logging.getLogger(__name__)
+
+DEFAULT_API_VERSION = '2'
+API_VERSION_OPTION = 'os_smn_api_version'
```

Comment:
maybe uppercase? if it's env var

Reply:
I keep the general format of the other services which was introduced by Artem. So I won't change it.